src/script/hydra-eval-jobset

```diff
@@ -644,7 +644,7 @@ sub checkJobsetWrapped {
     # Hash the arguments to hydra-eval-jobs and check the
     # JobsetInputHashes to see if the previous evaluation had the same
     # inputs.  If so, bail out.
-    my @args = ($jobset->nixexprinput, $jobset->nixexprpath, inputsToArgs($inputInfo));
+    my @args = ($jobset->nixexprinput // "", $jobset->nixexprpath // "", inputsToArgs($inputInfo));
```

Comment:
Should this maybe also take into account the flake-related arguments? Seems a bit strange to hash on things that are only defined for so-called "legacy" input types.

Reply:
Honestly I don't know, so I'm hesitant to do so :grimacing:.